README.md

```diff
@@ -198,6 +198,15 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ```
 
+## Getting Support
+
+The Autoscaler is a [Cloud Spanner Ecosystem](https://www.cloudspannerecosystem.dev/about)
+project based on open source contributions. We'd love for you to report
+issues, file feature requests, and send pull requests (see Contribution).
```

Comment:
```suggestion
project based on open source contributions. We'd love for you to 
[report issues, file feature requests][new-issue], and [send pull requests][new-pr] (see [Contributing](README.md#contributing)).
```

Also please add the following links at the very end of the file:
[new-issue]: https://github.com/cloudspannerecosystem/autoscaler/issues/new
[new-pr]: https://github.com/cloudspannerecosystem/autoscaler/compare